spec/cc/engine/analyzers/sexp_lines_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+require "cc/engine/duplication"
+
+module CC::Engine::Analyzers
+  RSpec.describe SexpLines do
+    let(:source) do
```

Comment:
WDYT about inlining this into the example where it's used? It's only used there, and it's important to the spec, so I think it should be co-located.